exporters/otlp/src/otlp_recordable_utils.cc

```diff
@@ -259,25 +259,19 @@ void OtlpRecordableUtils::PopulateRequest(
     return;
   }
 
-  auto resource_span       = request->add_resource_spans();
-  auto instrumentation_lib = resource_span->add_instrumentation_library_spans();
-  bool first_pass          = true;
-
   for (auto &recordable : spans)
   {
     auto rec = std::unique_ptr<OtlpRecordable>(static_cast<OtlpRecordable *>(recordable.release()));
+    auto resource_span       = request->add_resource_spans();
```

Comment:
Can resource_span be outside the loop? Otel-cpp doesn't support multiple TracerProviders, so the resources would be always the same for all the events.

Reply:
@lalitb, could otel-cpp create multiple `TracerProvider`, and set each of them as the global one in alternative?

Reply:
Yes possible. Have approved it as it was just a nit comment :) should be good to merge.